src/components/gauge/gauge.scss

```diff
@@ -0,0 +1,222 @@
+        & span {
+          width: 8px;
+          height: 8px;
+          transform: translateY(-90%) rotate(45deg) skew(12deg, 12deg);
+          z-index: 1;
+          background-color: var(--max-value-display-background);
+          position: absolute;
+        }
+      }
+    }
+
+    .minMaxDisplay {
+      position: absolute;
+      display: flex;
+      justify-content: space-between;
+      align-content: center;
+      width: 100%;
+      height: auto;
+      .maxValue {
+        display: flex;
+        justify-content: center;
+        align-content: center;
+        background-color: var(--max-value-display-background);
+        color: var(--max-value-display-color);
+        border-radius: 10px;
+        padding: 5px;
+        float: right;
+        & span {
+          width: 8px;
+          height: 8px;
+          transform: translate(200%, -100%) rotate(45deg) skew(12deg, 12deg);
+          z-index: -1;
+          background-color: var(--max-value-display-background);
+          position: absolute;
+          border-radius: 0% 200% 0% 10%;
+        }
+      }
+      .minValue {
+        display: flex;
+        justify-content: center;
+        align-content: center;
+        background-color: var(--min-value-display-background);
+        color: var(--min-value-display-color);
+        border-radius: 10px;
+        padding: 5px;
+        & span {
+          width: 8px;
+          height: 8px;
+          border-radius: 0% 10% 0% 200%;
+          transform: translate(calc(-100%), calc(-100%)) rotate(45deg)
+            skew(12deg, 12deg);
+          z-index: -1;
+          background-color: var(--min-value-display-background);
+          position: absolute;
+        }
+      }
+    }
+    .rangesContainer {
+      overflow: hidden;
+      border-radius: 20px;
+      position: absolute;
+      width: calc(100%);
+      height: 100%;
+      background-color: rgba(0, 0, 0, 0.1);
+      .range {
+        height: 100%;
+        position: absolute;
+        box-shadow: var(--gauge-ranges-box-shadow);
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
+    }
+    .namesContainer {
+      // position: absolute;
+      width: calc(100% + 4px);
+      height: calc(100% + 4px);
+      display: flex;
+      // justify-content: center;
+      align-items: center;
+      .rangeName {
+        height: 100%;
+        position: absolute;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        filter: brightness(0.5);
+        // transform: translateY(100%);
+      }
+    }
+  }
+  .svgContainer {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    .gaugeContainer {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      border-radius: 50%;
+      z-index: 0;
+      position: absolute;
+    }
+    svg {
+      z-index: -1;
+    }
+    .gauge {
+      border-radius: 50%;
+      font-size: 1.1rem;
```

Comment:
I'd avoid setting the font size and allow it to cascade to allow for more customization. What do you think?

Reply:
Mmm yes. You're right.
I thought it was a good idea to set a default font-size and remove responsibility of the developer.
I will fix that

Reply:
Check new changes about font-size in commit f67e266